tests/net/ethernet_mgmt/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  test:
+    min_ram: 32
+    tags: net ethernet_mgmt
```

Comment:
We could also add "depends_on: netif" here so that the test automatically runs on those boards that support ethernet.